### ISSUE TITLE
Member list text color

### ISSUE THREAD

**Issue by nfearnley** [MCMrARM/revolution-irc#207]
I am using a theme with a black background. Occasionally, when I pull out the member list, all the nicknames will have a text color of black. This is fixed by opening up the theme settings, the resaving the theme. Once this is done, the text color appears properly for a while. Eventually, the text color reverts to black again.

**Comment by MCMrARM** [comment]
Duplicate of https://github.com/MCMrARM/revolution-irc/issues/202
